qcore/constants.py

```diff
@@ -396,6 +398,16 @@ def component_sorter(x):
 
         return components_to_get, components_to_store
 
+    @staticmethod
+    def get_basic_components():
+        return [
+            Components.c090,
+            Components.c000,
+            Components.cver,
+            Components.ch1,
```

Comment:
I'm not convinced that ch1 is a basic component?

Reply:
The advanced ones are all derived, so it definitely doesn't fit with them

Reply:
If i'm not wrong, h1 h2 from definition has to be calculated after the waveform has been recorded?
its just happens to be in ascii formatting. 
If the word 'basic' become confusing, we can change the naming.
I personally doesnt have too much issue with this, but i can see why jason questioned it at the first place

Reply:
however, if we actually are considering the two as basic components, wouldn't it be better to insert them into the list instead of appending? just for classification.
like the first few is always basic components, and then comes the advanced
so we wont have random sequences of basic and advanced components.

Reply:
h1, h2 are the raw waveforms recorded at station. 000, 090 are rotated to the cardinal directions - North and East

It doesn't really matter the order of them, I generally prefer adding them in order of history.